src/locales/en/QueryPage.json

```diff
@@ -7,7 +7,7 @@
     "Results": {
         "Parameter": "Selected Filters",
         "TableHead": "Number of isolates",
-        "TableHeadRelative": "Isolates in %",
+        "TableHeadRelative": "Isolates",
```

Comment:
Reject: You forgot to rename TableHead and TableHeadRelative here as in the german file.

Reply:
Done. Thx!